l3build/l3build.lua

```diff
@@ -1383,29 +1383,31 @@ end
 --
 
 -- An auxiliary used to set up the environmental variables
-function runtool(envvar, command)
+function runtool(subdir, dir, envvar, command)
   return(
     run(
-      typesetdir,
+      typesetdir .. "/" .. subdir,
```

Comment:
As is is also the case with the previous comment about a trailing `/`, nearly all paths are of the form `./path/to/a/file`. So `path` should never end up empty to begin with, but at least be `.`. The parameter is then passed to `cd`, which also is not confused by a trailing `/`.  But a check for an empty `path` could well be added.